.github/ISSUE_TEMPLATE/bug-report-🐞.md

```diff
@@ -0,0 +1,31 @@
+---
+name: "Bug report \U0001F41E"
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Current behavior**
+A clear and concise description of what happens now.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**[Image version](https://github.com/GoogleCloudPlatform/khi/pkgs/container/khi) used**: 
+
+- [ ] I use the latest Chrome when I observed this issue
```

Comment:
Can we ask users to upload the `.khi` file if they can?

Reply:
Good idea, added